drafts/Stormless3.md

```diff
@@ -0,0 +1,322 @@
+
+# Episode 3
+
+
+Build a serverless app on AWS with [Serverless Framework][sf], AWS Lambda, StepFunctions, DynamoDB, API Gateway, and ready-to-use functions from StackStorm Exchange open-source catalog.
+
+[sf]: https://serverless.com/framework
+
+You are watching the Episode Three. 
+
+In the [Episode One][ep-one], I described the application we are building, walked you over setting up the development environment and creating Serverless project, and shown how to build a first Lambda function from [StackStorm Exchange][st2-ex] action with [Serverless Framework][sf].
+
+In the [Episode Two][ep-two], we added more actions: one native Lambda to record user info to DynamoDB, and another one from StackStorm Exchange to make a call to ActiveCampaign CRM system. You learned more of `serverless.yml` syntax and practiced developer workflow working with individual functions. 
+
+[ep-one]:https://medium.com/@dzimine/tutorial-building-a-community-on-boarding-app-with-serverless-stepfunctions-and-stackstorm-b2f7cf2cc419
```

Comment:
What are these? Seem like random links without context - hoping these will be placed elsewhere? The formatting is strange too.